src/algebra/star/self_adjoint.lean

```diff
@@ -152,3 +166,17 @@ instance : module R (self_adjoint A) :=
 end module
 
 end self_adjoint
+
+lemma is_star_normal_iff [monoid R] [star_monoid R] {x : R} :
+  is_star_normal x ↔ star x * x = x * star x :=
+iff.rfl
```

Comment:
Arguably this lemma isn't necessary, anywhere you want to use `rw is_star_normal_iff` you can just use `rw is_star_normal`